ci-operator/step-registry/origin/e2e-aws-sdn-single/origin-e2e-aws-sdn-single-workflow.yaml

```diff
@@ -0,0 +1,14 @@
+workflow:
+  as: origin-e2e-aws-sdn-single
+  steps:
+    pre:
+    - chain: ipi-conf-aws
+    - ref: sdn-conf
+    - ref: sdn-conf-single-manifest
+    - chain: ipi-install
+    test:
+    - ref: origin-e2e-test
+    post:
+    - chain: ipi-deprovision
+  documentation: |-
+    The Origin E2E SDN workflow executes the common end-to-end test suite with the OpenShiftSDN single tenant network plugin.
```

Comment:
nit - here in the documentation change "The Origin E2E SDN workflow" -> "The Origin E2E SDN single workflow" this will differentiate the doc string from the multi tenet case

Reply:
Changed.